### PR TITLE
120-skiplinks-b: Alternative render.

### PR DESCRIPTION
**Pull request by sherakama**
@JBCSU @yvonnetangsu @josephgknox 

How about something like this. With this approach, the twig template is usable directly in your work and the rendered output has the example we want. 

Thoughts?

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
> @JBCSU @yvonnetangsu @josephgknox
> 
> How about something like this. With this approach, the twig template is usable directly in your work and the rendered output has the example we want.
> 
> Thoughts?

I do like this approach! Btw, I just noticed this now, should we make this singular "skiplink" so it's consistent with all the other components?

File: core/scss/components/skiplinks/_skiplinks.scss [hunk 1]
```diff
@@ -10,7 +10,13 @@
 // to the main content as sighted, mouse users.
 // of the main content. Learn more at https://webaim.org/techniques/skipnav.
 //
-// Markup: ../../templates/components/skiplinks/skiplinks.twig
+// Markup: <header>
+//   <a href="#skip-to-content" class="su-skiplinks">Skip to content</a>
```
Comment (yvonnetangsu) [inline core/scss/components/skiplinks/_skiplinks.scss]:
```suggestion
//   <a href="#main-content" class="su-skiplinks">Skip to content</a>
```

File: core/scss/components/skiplinks/_skiplinks.scss [hunk 2]
```diff
@@ -10,7 +10,13 @@
...
+// </header>
+//  <main>
+//   <h2 id="skip-to-content" class="su-sr-only-text" tabindex="-1">Main Content</h2>
```
Comment (yvonnetangsu) [inline core/scss/components/skiplinks/_skiplinks.scss]:
```suggestion
//   <h2 id="main-content" class="su-sr-only-text" tabindex="-1">Main Content</h2>
```

**Comment by josephgknox** [comment]
@sherakama this approach also works well for me. I'll merge this into the other branch. We can continue discussion there if needed.

@yvonnetangsu it's a good question regarding the component naming convention. I'd argue that this accessibility-related component might be OK to leave as plural, as the chances of this not skipping more than one link (used primarily for skipping navigation) is rare. Also, it appears to most commonly be referred to as Skiplinks on the inter-webs. Willing to discuss this more, too. I might be crazy. :)

**Comment by yvonnetangsu** [comment]
@josephgknox I'm comfortable going with the most commonly known term 😸 Thanks for checking!

**Comment by JBCSU** [comment]
@sherakama @josephgknox I just committed to this branch an alternative that I think works quite well. Please let me know what you think.

**Pull request merged**